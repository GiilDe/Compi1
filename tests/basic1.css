/* this
 is a multiline
 comment */

 /*so*is*this*/

helloWorld;
-dash_underscore;
-dashing----thru11-;

#deadbeef

#123456_hello

#-There

#124604;

@import ("lib.css");

!important;

! iMpOrTaNt;

!
  important;

"simple";

'also simple';

"escape new lines\n";

'"hex"\0045a';
'\011111HELLO';
"\01 'h'hello";

table + all {
  attribute: 80%;
}
table ~ div#first img:hover
table.colorful > input[type="text"]
