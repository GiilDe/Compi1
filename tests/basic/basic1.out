3 COMMENT 3
5 COMMENT 1
7 NAME helloWorld
7 SEMICOLON ;
8 NAME -dash_underscore
8 SEMICOLON ;
9 NAME -dashing----thru11-
9 SEMICOLON ;
11 HASHID #deadbeef
13 HASHID #123456_hello
15 HASHID #-There
17 HASHID #124604
17 SEMICOLON ;
19 IMPORT @import
19 STRING lib.css
19 SEMICOLON ;
21 IMPORTANT !important
21 SEMICOLON ;
23 IMPORTANT ! iMpOrTaNt
23 SEMICOLON ;
26 IMPORTANT !
  important
26 SEMICOLON ;
28 STRING simple
28 SEMICOLON ;
30 STRING also simple
30 SEMICOLON ;
32 STRING escape new lines\n
32 SEMICOLON ;
34 STRING "hex"
34 SEMICOLON ;
35 STRING HELLO
35 SEMICOLON ;
36 STRING \01 'h'hello
36 SEMICOLON ;
38 NAME table
38 COMB +
38 NAME all
38 LBRACE {
39 NAME attribute
39 COLON :
39 UNIT 80%
39 SEMICOLON ;
40 RBRACE }
41 NAME table
41 COMB ~
41 NAME div
41 HASHID #first
41 NAME img
41 COLON :
41 NAME hover
42 NAME table
42 DOT .
42 NAME colorful
42 COMB >
42 NAME input
42 LBRACKET [
42 NAME type
42 EQUAL =
42 STRING text
42 RBRACKET ]
